client/src/game/ui/HanabiCard.ts

```diff
@@ -462,29 +429,33 @@ export default class HanabiCard extends Konva.Group {
     // If the card is already fully revealed from clues, then additional clues would tell us nothing
     // We don't check for "this.identityDetermined" here because we still need to calculate the
     // effects of clues on cards in other people's hands that we already know the true identity of
-    const wasFullyKnown = this.possibleSuits.length === 1 && this.possibleRanks.length === 1;
+    const wasFullyKnown = this.state.possibleSuits.length === 1
```

Comment:
breaks airbnb style, same as previous thing

Reply:
The current linter options doesn't allow
```
const wasFullyKnown = 
  this.state.possibleSuits.length === 1
```

Reply:
Maybe with a paren. I'll check